### PR TITLE
feat: add new secondary school survey message 🐿 v2.12.5

### PR DESCRIPTION
**Pull request by alexnaish**
Due to the need to change back to the old message on interaction with the survey, its easier to treat this as an alternative SSI message - targeted the same but higher priority message and strict rules about interaction. (TBD in Ammit)

Screenshot 2020-01-31 at 11 23 36

### CODE REVIEW

File: templates/partials/bottom/secondary-school-survey.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{{#> n-messaging-client/templates/components/o-banner
+	themeMarketing="true"
+	small="true"
+}}
+	<div class="o-banner__inner" data-o-banner-inner="" data-o-banner-inner="">
```
Comment (umbobabo) [inline templates/partials/bottom/secondary-school-survey.html]: Duplicate empty data attribute 🤔

File: templates/partials/bottom/secondary-school-survey.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		<div class="o-banner__content o-banner__content--short">
+			<header class="o-banner__heading n-messaging-banner__heading">
+				<h1>How do you rate FT Schools?</h1>
+			</header>
+			<p>We want your feedback on the FT so we can be more relevant and useful to young people.</p>
+		</div>
+		<!-- Button and link -->
+		<div class="o-banner__actions">
+			<div class="o-banner__action">
+				<a class="o-banner__button" href="https://survey.ft.com/jfe/form/SV_6YkvMOTIgDoRgb3" target="_blank" data-o-banner-action="" role="link">
```
Comment (umbobabo) [inline templates/partials/bottom/secondary-school-survey.html]: Is it `data-o-banner-action=""` expected to be empty?

File: templates/partials/bottom/secondary-school-survey.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+					Give your feedback on FT Schools
+				</a>
+			</div>
+		</div>
+
```
Comment (umbobabo) [inline templates/partials/bottom/secondary-school-survey.html]:
```suggestion

```

File: templates/partials/bottom/secondary-school-survey.html (hunk 1 above)
Comment (alexnaish) [reply templates/partials/bottom/secondary-school-survey.html]: Will remove but its duplicated like this in every component template for some reason!

File: templates/partials/bottom/secondary-school-survey.html (hunk 2 above)
Comment (alexnaish) [reply templates/partials/bottom/secondary-school-survey.html]: This is something all templates have - will try to see if removing changes anything on analytics

**Pull request merged**